business/src/main/resources/org/mskcc/cbio/portal/persistence/StudyMapperLegacy.xml

```diff
@@ -14,7 +14,7 @@
         DESCRIPTION as description,
         PMID as pmid,
         CITATION as citation,
-        GROUPS as groups,
+        "GROUPS" as "groups",
```

Comment:
Why is this put here and if required, should it be added to getAllStudies in the same file?

Reply:
We should always quote those mySql reserved key words. I agree with you any fixing should not be in this PR. Will undo the change.